### PR TITLE
Userstory7

### PR DESCRIPTION
**Pull request by DarkoKrezic**
[US7](https://github.com/DarkoKrezic/capstone-projekt/issues/13)

### CODE REVIEW

File: components/StoryList/StyledStoryList.js [hunk 1]
```diff
@@ -55,9 +55,9 @@ export const StoryListContainer = styled.ul`
 
 export const StoryListCard = styled.div`
   display: flex;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  /* border: 1px solid #ccc;
+  border-radius: 4px; */
+  /* box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1); */
```
Comment (doctype-Melvin) [inline components/StoryList/StyledStoryList.js]: If your code doesn't need these lines you can remove them. As far as I know we're not supposed to keep muted code.

File: components/UseStorytellerForm/Index.js [hunk 2]
```diff
@@ -1,58 +1,53 @@
...
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(storyObjectPrompt),
+      });
+      if (!response.ok) {
+        throw new Error("Failed to create story object");
+      }
+      const newGeneratedStory = await response.json();
+      console.log(newGeneratedStory);
```
Comment (doctype-Melvin) [inline components/UseStorytellerForm/Index.js]: I think we shouldn't keep console.logs in our code when making PRs.

File: pages/api/GeneratedStory.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import { v4 as uuidv4 } from "uuid";
+
+const configuration = new Configuration({
+  organization: "org-KWKX4sUxoLRs3G6H2PQOR6oN",
+  apiKey: process.env.OPENAI_API_KEY,
+});
+const openai = new OpenAIApi(configuration);
+
+async function generateCoverImage(coverImagePrompt) {
+  //   console.log(coverImagePrompt);
```
Comment (doctype-Melvin) [inline pages/api/GeneratedStory.js]: You can remove this line.

**Review by doctype-Melvin — CHANGES_REQUESTED** [review]
Hi Darko!

I think you're making great progress. Just left some comments about minor things. 
Maybe a coach will look through your updated PR for any additional suggestions. 

So far yo

File: components/StoryList/StyledStoryList.js (hunk 1 above)
Comment (DarkoKrezic) [reply components/StoryList/StyledStoryList.js]: You are absolutely right! Will be removed

File: components/UseStorytellerForm/Index.js (hunk 2 above)
Comment (DarkoKrezic) [reply components/UseStorytellerForm/Index.js]: Here as well

File: pages/api/GeneratedStory.js (hunk 3 above)
Comment (DarkoKrezic) [reply pages/api/GeneratedStory.js]: Will be changed before the Lunchbreak is over 😅🙈

File: styles.js [hunk 4]
```diff
@@ -6,12 +6,16 @@ export default createGlobalStyle`
   *::after {
     box-sizing: border-box;
   }
-
+  /* @font-face {
```
Comment (onemanwenttomow) [inline styles.js]: I don't think we want commented out code making it to `main`

**Review by onemanwenttomow — APPROVED** [review]
Nice job @DarkoKrezic  🎉

One little bit of commented out code to remove but otherwise looks good.

**Pull request merged**